StripeConnect/StripeConnectTests/Internal/Webview/ConnectWebViewTests.swift

```diff
@@ -49,7 +54,8 @@ class ConnectWebViewTests: XCTestCase {
         wait(for: [expectation], timeout: TestHelpers.defaultTimeout)
     }
 
-    func testOpenSafariVCIfNotInAllowedHosts() {
+    /// HTTP/HTTPS navigations with no target from a non-allowlisted host should open in a SafariVC
+    func testPopupOpenSafariVCIfNotInAllowedHosts() {
```

Comment:
Added 'Popup' to these existing tests since I added very similar ones for non-popup responses